@BabyKangaroo117 

I ran the instructions within testing guidelines and ran into the following error:
```
# YARTBML/lexer
lexer/lexer_test.go:108:11: missing ',' in composite literal
FAIL    YARTBML/lexer [setup failed]
FAIL
```

--- a/src/lexer/lexer.go
+++ b/src/lexer/lexer.go
@@ -15,17 +15,15 @@ func New(input string) *Lexer {
 	return l
 }
 
-
 func (l *Lexer) readChar() {
 	if l.readPosition >= len(l.input) {
-	l.ch = 0
+		l.ch = 0
 	} else {
-	l.ch = l.input[l.readPosition]
+		l.ch = l.input[l.readPosition]
 	}
 	l.position = l.readPosition
 	l.readPosition += 1
 }
-	
 
 func newToken(tokenType token.TokenType, ch byte) token.Token {
 	return token.Token{Type: tokenType, Literal: string(ch)}
@@ -135,5 +133,3 @@ func (l *Lexer) peekChar() byte {
 	}
 }
 
-
-	

--- a/src/lexer/lexer_test.go
+++ b/src/lexer/lexer_test.go
@@ -104,9 +104,6 @@ func TestNextToken(t *testing.T) {
 		{token.INT, "9"},
 		{token.SEMICOLON, ";"},
 		{token.EOF, ""},
-
-		10 == 10;
-		10 != 9;
 	}
 	
 	l := New(input)

--- a/src/lexer/lexer_test.go
+++ b/src/lexer/lexer_test.go
@@ -6,8 +6,7 @@ import (
 )
 
 func TestNextToken(t *testing.T) {
-	input := 
-		`
+	input := `
 		let five = 5;
 		let ten = 10;
 		let add = fn(x, y) {
@@ -27,8 +26,8 @@ func TestNextToken(t *testing.T) {
 		10 != 9;
 		`
 	tests := []struct {
-	expectedType token.TokenType
-	expectedLiteral string
+		expectedType    token.TokenType
+		expectedLiteral string
 	}{
 		{token.LET, "let"},
 		{token.IDENT, "five"},
@@ -66,9 +65,9 @@ func TestNextToken(t *testing.T) {
 		{token.IDENT, "ten"},
 		{token.RPAREN, ")"},
 		{token.SEMICOLON, ";"},
-		{token.BANG, "!"}, 
+		{token.BANG, "!"},
 		{token.MINUS, "-"},
-		{token.SLASH, "/"}, 
+		{token.SLASH, "/"},
 		{token.ASTERISK, "*"},
 		{token.INT, "5"},
 		{token.SEMICOLON, ";"},
@@ -76,10 +75,10 @@ func TestNextToken(t *testing.T) {
 		{token.LT, "<"},
 		{token.INT, "10"},
 		{token.GT, ">"},
-		{token.INT, "5"}, 
+		{token.INT, "5"},
 		{token.SEMICOLON, ";"},
 		{token.IF, "if"},
-		{token.LPAREN, "("}, 
+		{token.LPAREN, "("},
 		{token.INT, "5"},
 		{token.LT, "<"},
 		{token.INT, "10"},
@@ -105,7 +104,7 @@ func TestNextToken(t *testing.T) {
 		{token.SEMICOLON, ";"},
 		{token.EOF, ""},
 	}
-	
+
 	l := New(input)
 
 	for i, tt := range tests {
@@ -121,4 +120,3 @@ func TestNextToken(t *testing.T) {
 		}
 	}
 }
-

--- a/src/main.go
+++ b/src/main.go
@@ -1,10 +1,10 @@
 package main
 
 import (
+	"YARTBML/repl"
 	"fmt"
 	"os"
 	"os/user"
-	"YARTBML/repl"
 )
 
 func main() {
@@ -12,8 +12,8 @@ func main() {
 	if err != nil {
 		panic(err)
 	}
-		fmt.Printf("Hello %s! This is the Monkey programming language!\n",
+	fmt.Printf("Hello %s! This is the YARTBML programming language!\n",
 		user.Username)
-		fmt.Printf("Feel free to type in commands\n")
-		repl.Start(os.Stdin, os.Stdout)
+	fmt.Printf("Feel free to type in commands\n")
+	repl.Start(os.Stdin, os.Stdout)
 }

--- a/src/repl/repl.go
+++ b/src/repl/repl.go
@@ -1,14 +1,15 @@
 package repl
 
 import (
+	"YARTBML/lexer"
+	"YARTBML/token"
 	"bufio"
 	"fmt"
 	"io"
-	"YARTBML/lexer"
-	"YARTBML/token"
 )
 
 const PROMPT = ">> "
+
 func Start(in io.Reader, out io.Writer) {
 	scanner := bufio.NewScanner(in)
 	for {
@@ -24,3 +25,4 @@ func Start(in io.Reader, out io.Writer) {
 		}
 	}
 }
+

--- a/src/token/token.go
+++ b/src/token/token.go
@@ -42,21 +42,20 @@ const (
 	// Keywords
 	FUNCTION = "FUNCTION"
 	LET      = "LET"
-	TRUE = "TRUE"
-	FALSE = "FALSE"
-	IF = "IF"
-	ELSE = "ELSE"
-	RETURN = "RETURN"
-
+	TRUE     = "TRUE"
+	FALSE    = "FALSE"
+	IF       = "IF"
+	ELSE     = "ELSE"
+	RETURN   = "RETURN"
 )
 
 var keywords = map[string]TokenType{
-	"fn": FUNCTION,
-	"let": LET,
-	"true": TRUE,
-	"false": FALSE,
-	"if": IF,
-	"else": ELSE,
+	"fn":     FUNCTION,
+	"let":    LET,
+	"true":   TRUE,
+	"false":  FALSE,
+	"if":     IF,
+	"else":   ELSE,
 	"return": RETURN,
 }
 
@@ -66,3 +65,4 @@ func LookupIdent(ident string) TokenType {
 	}
 	return IDENT
 }
+

